projects/linux/Makefile

```diff
@@ -24,10 +24,7 @@ INCLUDES=					\
 
 EXTRA_CFLAGS +=-Wall
 EXTRA_CFLAGS +=-Werror
-EXTRA_CFLAGS +=-Wno-switch-break
-EXTRA_CFLAGS +=-Wno-pointer-to-int-cast
```

Comment:
My take is that the -Wpointer-to-int-cast is a appopriate warning and it is a pity to hide it (so other possible misuse) besause of one (legal) use. It is a bit surprising you cannot do without it. But this is your call.